### DISCUSSION TITLE
nodelete C++ JNI files

### DISCUSSION THREAD

**Discussion by oOTWK** [bytedeco/javacpp-presets#1247] [Q&A] [answered]
I want to have C++ JNI files undeleted and found that javacpp.jar takes `-nodelete` option.
How can I pass this option in javacpp-presets maven?

**Comment by saudet** [comment, marked as answer]
Set "deleteJniFiles" to "false":
http://bytedeco.org/javacpp/apidocs/org/bytedeco/javacpp/tools/BuildMojo.html#deleteJniFiles

**Reply by oOTWK** [reply]
Thank you!
